website_hr_contact/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===============
+HR Address Book
+===============
+
+This module add a new page into your website displaying the address book of your employees.
+The informations related to your employees are publicly available if your employee is
+published. The same rule applies to the address informations.
+
+Usage
+=====
```

Comment:
Please add usage instructions, to know what to test.

Reply:
@JonathanNEMRY ping :)